phpunit.xml.dist

```diff
@@ -264,21 +264,21 @@
 		<directory>tests/Backend/Exasol</directory>
 	</testsuite>
 
-	<testsuite name="paratest-teradata">
+	<testsuite name="paratest-teradata-part-1">
 
 		<directory>tests/Backend/CommonPart1</directory>
 		<exclude>tests/Backend/CommonPart1/TimeTravelTest.php</exclude>
 		<!--	excluded test is only for BQ specific -->
 		<exclude>tests/Backend/CommonPart1/BigqueryTableExporterTest.php</exclude>
 		<!--		TD does not support TWC-->
+		<exclude>tests/Backend/CommonPart1/TableWithConfigurationLoadFromWorkspaceTest.php</exclude>
+		<exclude>tests/Backend/CommonPart1/TableWithConfigurationLoadTest.php</exclude>
 		<exclude>tests/Backend/CommonPart1/CreateTableWithConfigurationTest.php</exclude>
 		<exclude>tests/Backend/CommonPart1/MigrateTableWithConfigurationTest.php</exclude>
-		<exclude>tests/Backend/CommonPart1/RegisterBucketTest.php</exclude>
+	</testsuite>
 
+	<testsuite name="paratest-teradata-part-2">
```

Comment:
A dává to smysl rozdělené takhle? Jakože všechny ostatní backendy mají CommonPart2 v part1 a v part2 mají něco jiného. (jakkoli to zní nesmyslně)...

Reply:
Jako třea SNFLK P2:
```
		<file>tests/Backend/CommonPart1/BucketsTest.php</file>
		<file>tests/Backend/Workspaces/WorkspacesLoadTestReadOnly.php</file>
		<file>tests/Backend/Workspaces/ComponentsWorkspacesTest.php</file>
		<file>tests/Backend/Snowflake/CloneIntoWorkspaceTest.php</file>
		<file>tests/Backend/Snowflake/CopyIntoWorkspaceTest.php</file>
		<file>tests/Backend/Export/ExportParamsTest.php</file>
		<file>tests/Backend/ExternalBuckets/SnowflakeRegisterBucketTest.php</file>
```

Reply:
Aha, to me ani nenapadlo, ze by to mohlo byt jinak :D anyway, cilem tohoto je rozmelnit zatez. v tomto nastaveni ma `paratest-teradata-abs-part-1` 146 casu, `paratest-teradata-abs-part-2` 124 casu, takze za me je to dost dobre rozdeleni. plus to semanticky dava smysl (jako vyjimka :D )

Neni to ale bulletproof, stale ty testy nahodne padaji, zejmena na TPT -> to bych chtel nejak experimentovat v https://keboola.atlassian.net/browse/TER-106 . Jedno z reseni je rozdelit to jeste vice, ale na to potrebujeme vice masin